app/views/quotes/update.turbo_stream.erb

```diff
@@ -0,0 +1,3 @@
+<%= turbo_stream.replace @quote do %>
+  <%= render partial: 'quotes/quote', locals: { quote: @quote }%>
+<% end %>
```

Comment:
4.- I had to create the update turbo stream file to get the ID already updated. As a result, I can get the data in the template index. 

Only after doing all this I realize that in the tutorial it was not necessary to do the turbo stream configuration, instead they just add their IDs in each 'quote' and Rails automatically detects that this ID will be replaced by updating, I don't know why it can't do it without turbo stream